x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/310_rrf_retriever_simplified.yml

```diff
@@ -429,3 +482,49 @@ setup:
   - match: { hits.hits.0._id: "2" }
   - match: { hits.hits.1._id: "1" }
   - match: { hits.hits.2._id: "3" }
+
+
+---
+"Queries multiple indices using default_field":
+  - requires:
+      cluster_features: [ "rrf_retriever.multi_index_simplified_format_support" ]
+      reason: "Support for querying multiple indices in simplified RRF retriever"
+  - do:
+      search:
+        index: test-index,test-another-index
+        body:
+          retriever:
+            rrf:
+              query: "match"
+
+  - match: { hits.total.value: 6 }
+  - length: { hits.hits: 6 }
+  - match: { hits.hits.0._id: "4" }
```

Comment:
Have we tested this with different shard counts, to make sure it won't be flakey in serverless where the default shard count is different or with different randomized shard configurations?